### PR TITLE
Adds tests to amp handler

### PR DESCRIPTION
**Pull request by shraddha-kesari**

### CODE REVIEW

File: server/handlers/amp-handler.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ exports.handleAmpRequest = async function handleAmpRequest(
       "amp-config",
       async () => await AmpConfig.getAmpConfig(client)
     );
-    const slug = Number(0);
+    const slug = String(0);
     const story = await Story.getStoryBySlug(client, req.params[slug]);
```
Comment (ags1773) [inline server/handlers/amp-handler.js:33]: Can directly do `req.params["0"]`

File: test/integration/amp-handler-test.js [hunk 2]
```diff
@@ -104,9 +104,21 @@ function createApp(app = express()) {
...
+    publisherConfig: {},
+    ampLibrary,
+  });
+  return app;
+}
+
 describe("AmpHandler", () => {
   it("mounts an amp page", (done) => {
-    const app = createApp();
+    const app = createHomePageApp();
```
Comment (ags1773) [inline test/integration/amp-handler-test.js:121]: Don't need two apps, you can test both routes on the same app

Comment (shraddha-kesari) [reply test/integration/amp-handler-test.js:121]: okay.

File: server/handlers/amp-handler.js (hunk 1 above)
Comment (shraddha-kesari) [reply server/handlers/amp-handler.js:33]: Thought this would be more clean.

**Pull request merged**